Source/Editor/DualityEditor/Forms/MainForm.cs

```diff
@@ -501,6 +504,36 @@ private void actionRunApp_Click(object sender, EventArgs e)
 			AppRunningDialog runningDialog = new AppRunningDialog(appProc);
 			runningDialog.ShowDialog(this);
 		}
+		public void SaveDockPanelData(DualityEditorUserData dualityEditorUserData)
+		{
+			using (var str = new MemoryStream())
```

Comment:
I know this is a micro-manage comment, but can we name this variable something more descriptive? Even `stream` would be fine, but no need to shorten that even more.